tests/integration/targets/idrac_secure_boot/tests/invalid_common.yaml

```diff
@@ -78,7 +78,7 @@
           - invalid_ca_path.failed
           - not invalid_ca_path.changed
           - 'invalid_ca_path.msg == "[Errno 2] No such file or
-            directory: b''invalid''"'
+            directory"'
```

Comment:
@Saksham-Nautiyal, Kindly check this change

Reply:
Yes, Ansible is throwing this error msg only.